packages/docs/src/styles/pages/layout.scss

```diff
@@ -16,26 +16,13 @@ body {
   }
 }
 
-.page-content {
+.page-body {
   flex: 1 0 auto;
   max-width: 81rem;
 }
 
 @media (min-width: $media-width-lg) {
-  .page-content__content {
-    // removing margin for first heading on content pages
-    > :nth-child(2) {
-      margin-top: 0;
-    }
-    > h2 > h3,
-    > h4,
-    > h5,
-    > h6,
-    > p,
-    ul,
-    ol {
-      margin-bottom: 0;
-    }
```

Comment:
This is actually handled by the CSS reset in the design system now, and it works because there's no longer a mobile table of contents menu taking up the first-element position.